Interview/AutomationExercise.md

```diff
@@ -0,0 +1,70 @@
+<p align="center">
+<img src="../logo.png">
+</p>
+
+#Babylon 
+# iOS Automation engineer technical exercise
+
+At babylon, we firmly believe that **transparency** is a core value that should be present in everything we do, and hiring the best person should reflect that belief. That is why defined a clear and open technical exercise to be completed by candidates applying for an iOS automation engineer role. 
+
+## Our expectations
+
+Babylon aims to hire only the most qualified people. That doesn't mean that a candidate must have several academic degrees or years of experience. We value **potential** equally as much as **knowledge** and **experience**. Above all, we value one's ability to **think independently**, **solve problems** and to **engage with colleagues**.
+
+As such, these are the main areas the team will focus our assessment of candidates on:
+
+- Technical expertise
+   - Languages (Swift, Java) and platforms (iOS)
+   - Frameworks (UI testing with XCTest)
+   - Knowledge of software testing practises (Manual and Automated)
+   - Automation Architectural design patterns (Page Object Model, Cucumber)
+   - Knowledge of iOS APIs
+   - Knowledge of the development cycle
+- Interest in sharing one's knowledge
+- Interest in learning new knowledge
+- Interest in keeping oneself up to date
+- Ability to think and work independently
+- Ability to work in a large team
+- Professional and personal maturity
+- Overall attitude
+- Likelihood of fitting in well with the company's culture
+- Is the candidate someone we would like to work with on a daily basis?
+
+## The Technical Exercise
+
+### Application
+
+To begin with please download the following open source [application](https://github.com/khoren93/SwiftHub) and setup your environment. This will demonstrate your understanding of Xcode and building iOS application.
+
+### Existing Tests
+
+The project currently contains a number of UI test cases, for these tests we would like you to complete the following
+
+- Add a comment to each test outlining what is being tested
+- Describe if you feel the test is appropriate or well written
+- Improve the test if you feel it requires improvement
+- Verify the state of the test
+
+### New Test Cases
+
+For the net stage we would like you to add new tests to improve coverage
+
+- Create any new tests you feel are appropriate
+- Write a explanation as to why the test is needed and what it covers
+
+### Framework
+
+For the final stage we would like to understand your knowledge of how UI automation works on iOS and how tests interact with the application
+
+- Make any improvements you feel are appropriate to the following and clearly stating why:
+ - Update the test framework
```

Comment:
Shouldn't we state what would be the purpose to improve Test framework?

Reply:
In essence the purpose is to see if they are willing to add accessibility identifiers to the code to help make the automation more stable. But I did not want to give them a hint as to what we are looking for.

Reply:
So can we say:
```suggestion
 - Update the test framework to improve automation tests stability
```
Or do you feel this is giving too much information?